### PR TITLE
Minor fixes of wrong field names

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

**Comment by lego963** [comment]
[Add or update team repository permissions](https://docs.github.com/en/enterprise-server@3.2/rest/reference/teams#add-or-update-team-repository-permissions)

[Add a repository collaborator](https://docs.github.com/en/enterprise-server@3.2/rest/reference/collaborators#add-a-repository-collaborator)

File: plugins/modules/github_org_repository.py [hunk 1]
```diff
@@ -169,35 +168,39 @@
...
+        default: push
   collaborators:
     description: |
       Repository collaborators with their permissions
     type: list
     elements: dict
     options:
-      slug:
-        description: Team slug
+      username:
```
Comment (gtema) [inline plugins/modules/github_org_repository.py:178]: weird

**Review by gtema — CHANGES_REQUESTED** [review]

File: plugins/modules/github_org_repository.py (hunk 1 above)
Comment (lego963) [reply plugins/modules/github_org_repository.py:178]: doc was a little bit wrong

File: plugins/modules/github_org_repository.py [hunk 2]
```diff
@@ -233,7 +236,7 @@
                 description: |
                   The name of the required check.
                 type: str
-              appid:
+              app_id:
```
[the remark this answers is not in the document]
Comment (lego963) [reply plugins/modules/github_org_repository.py:239]: this is just a misstake in a doc

[the remark this answers is not in the document]
Comment (lego963) [reply plugins/modules/github_org_repository.py:239]: this is a mistake in doc, it's ok :)

File: plugins/modules/github_org_repository.py (hunk 1 above)
Comment (gtema) [reply plugins/modules/github_org_repository.py:178]: weird that sanity didn't complain. Is it running properly at all?

**Review by gtema — APPROVED** [review]

**Comment by gtema** [comment]
Yeah, we actually miss ansible-test job as such. Maybe try adding "- ansible-collection-test-sanity" and see whether it finds something else?

**Review by gtema — COMMENTED** [review]
I actually think spaces should stay. At least we should have a look in the rendered doc

**Review by gtema — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Pull request merged**